src/data_handler.py

```diff
@@ -0,0 +1,160 @@
+#import BoardInterface
+
+from enum import Enum
+
+class Sensor():
+
+    def __init__(self, sensorID, boardID, name):
+        self.sensorID = sensorID
+        self.boardID = boardID
+        self._lastValue = 0.0
+        self.name = name
+
+    def getSensorValue():
+        return _lastValue
+
+    def setSensorValue(sensorVal):
+        _lastValue = sensorVal
+
+    def getName():
+        return name
+
+    def _rawToRealValue(rawValue):
+        pass
+
+    def getSensorID():
+        return sensorID
+
+    def getBoardID():
+        return boardID
+
+class PressureSensor(Sensor):
+
+    def _rawToRealValue(rawValue):          #take in as hex value.
+        data = int(rawValue, 16)
+        return data * (1500 / 4095)         #Maps from (0x000,0xFFF) to (0,1500) psi
+
+class TempSensor(Sensor):
+
+    def _rawToRealValue(rawValue):
+        pass #bin dec conversion
+
+class ForceSensor(Sensor):
+
+    def _rawToRealValue(rawValue):
+        pass
+
+
+
+class Valve:
+
+    def __init__(self, valveID, boardID, closedAngle, openAngle, name):
+        self._valveID = valveID
+        self.boardID = boardID
+        self._closeAngle = closeAngle
+        self._openAngle = openAngle
+        self._lastValue = 0.0
```

Comment:
So, it turns out that we aren't actually going to have sensors that read the valve states. I'd say go ahead and remove the lastValue/getValue/rawToReal stuff for now - sorry :(